scalameta/scalameta/shared/src/test/scala/scala/meta/tests/quasiquotes/SuccessSuite.scala

```diff
@@ -851,7 +851,6 @@ class SuccessSuite extends FunSuite {
   }
 
   test("1 t\"$tpe forSome { ..$stats }\"") {
-    import scala.language.existentials
```

Comment:
Scalafix piggybacks entirely on "Unused import" warnings from scalac so false negatives/positivies are bugs in scalac.